starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/MavenGradleEnterprise.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.build.gradle;
+
+import com.fizzed.rocker.RockerModel;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.feature.build.maven.templates.extensions;
+import io.micronaut.starter.template.RockerTemplate;
+import jakarta.inject.Singleton;
+
+@Singleton
```

Comment:
I would prefer to have this logic inside the `GradleEnterprise` feature with protected methods so that they could be overriden instead of having an extra `@Singleton`

Reply:
I have removed `MavenGradleEnterprise`here: 

https://github.com/micronaut-projects/micronaut-starter/pull/1595/commits/c9b28bc5df6d7688dad2cf9e65e189832d2e7be3

@alextu let me know if you are ok with this change?

Reply:
Sure, no problem